### DISCUSSION TITLE
“开源”之后，会发生什么？

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#72] [资料分享]
在 DebCon2020 （Debian 开发者大会）上，Bruce Perens 就做过这样一个演讲，将开源过去和现在的问题剖析了出来，意图开启“后开源时代”。

https://www.youtube.com/watch?v=vTsc1m78BUk

![image](https://user-images.githubusercontent.com/15010826/163936102-c7ae1f3b-2718-4f9a-af30-4b5b8d71c8ef.png)

中文版资料：https://my.oschina.net/oscpyaqxylk/blog/5515202

**Comment by wangleo61** [comment]
好东东呀

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Tuesday, April 19, 2022 2:00 PM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] “开源”之后，会发生什么？ (Discussion #72)


在 DebCon2020 （Debian 开发者大会）上，Bruce Perens 就做过这样一个演讲，将开源过去和现在的问题剖析了出来，意图开启“后开源时代”。

https://www.youtube.com/watch?v=vTsc1m78BUk

[image]<https://user-images.githubusercontent.com/15010826/163936102-c7ae1f3b-2718-4f9a-af30-4b5b8d71c8ef.png>

中文版资料：https://my.oschina.net/oscpyaqxylk/blog/5515202

―
